api/repo/repair.go

```diff
@@ -116,6 +116,34 @@ func RepairRepo(c *gin.Context) {
 		}
 	}
 
+	// get repo information from the source
+	sourceRepo, err := scm.FromContext(c).GetRepo(ctx, u, r)
```

Comment:
just to clarify here.. this will follow GH's redirect for a renamed repo and return the new info? just asking since you are passing in current repo to fetch?

Reply:
Yes that is correct! It will follow GH's redirect for a renamed repo and uses the new info to update the current one in the DB